includes/wc-template-functions.php

```diff
@@ -686,36 +686,6 @@ function woocommerce_show_product_loop_sale_flash() {
 	}
 }
 
-if ( ! function_exists( 'woocommerce_get_product_schema' ) ) {
-
-	/**
-	 * Get a products Schema.
-	 * @return string
-	 */
-	function woocommerce_get_product_schema() {
```

Comment:
This cannot be removed - it will cause errors. It should be moved to the deprecated functions file - doesn't need to do anything though.